### PR TITLE
Update to latest Node 14

### PR DESCRIPTION
**Pull request by kraftbj**
Fixes https://nodejs.org/en/blog/vulnerability/july-2021-security-releases/

#### Changes proposed in this Pull Request:

* Updates to demand latest version of Node 14 to ensure everyone has updates versions and match Calypso (ref: https://github.com/Automattic/wp-calypso/pull/54275 )

#### Jetpack product discussion

p1625457364207800-slack-CBG1CP4EN (conversation re Calypso updating now)

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* `jetpack build plugins/jetpack` -- does it build?
*

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-14.16.1
+14.17.2
```
Comment (jeherve) [inline .nvmrc:1]: Looks like you'll need to update a few more places, like `.github/versions.sh`.

**Review by jeherve — CHANGES_REQUESTED** [review]

**Review by jeherve — DISMISSED** [review]
This looks good to me. 

@anomiex Is `v14.17.2` available to you?

**Comment by anomiex** [comment]
No, Debian's experimental package is still 14.17.0 at the moment. I'm starting to wonder if I should give up on using the distro package and start using nvm or volta or something, since I'm already using the package from experimental anyway.

One possibility so we don't have to wait on me to do that is that we could separate the "match Calypso" (i.e. update .nvmrc and versions.sh) and "force everyone to update" (i.e. update the range in package.json) parts of the PR.

**Comment by scinos** [comment]
FYI, Calypso is jumping to 14.17.3 (https://github.com/Automattic/wp-calypso/pull/54373), as there was some problem with CircleCI images that forced us to revert the 14.17.2 update (https://github.com/Automattic/wp-calypso/pull/54275#issuecomment-875252037)

**Review by jeherve — DISMISSED** [review]
Looks good to me. 👍

**Comment by kraftbj** [comment]
@anomiex What say you? Should I split things or are you game for using `nvm`?

**Comment by anomiex** [comment]
Depends on how soon you want this merged. I'll need to find time to figure out and set up a solution.

**Comment by kraftbj** [comment]
@anomiex I'm in no particular rush. I'd like to keep things up to date, but anyone using Calypso would have already upgraded and that setup should still work for us.

The updates between major versions are more critical to figure out a timeline.

**Comment by kraftbj** [comment]
Superseded by #20827

**Pull request closed without merging**